packages/manager/apps/dedicated/package.json

```diff
@@ -143,7 +143,6 @@
     "whatwg-fetch": "^3.0.0"
   },
   "devDependencies": {
-    "@ovh-ux/manager-models": "^0.0.0 || ^1.0.0",
```

Comment:
There already is a declaration in the `dependencies`

Reply:
If we only  use `import { foo } from @ovh-ux/manager-models` maybe we can leave it in the `devDependencies`

Reply:
Don't know, it's in the `dependencies` everywhere, you shouldn't have to worry about how a package is used to know how you declare it, or the day you change the way you use it, then you have to change the way you declare it 🤔

Reply:
@marie-j The business logic contained inside the `@ovh-ux/manager-models` module looks like a dependency from the Dedicated application point of view, so I'm ok with theses changes.

Is that make sense for you?

Reply:
@FredericEspiau indeed, however in the current context there is a difference between `devDependencies` and `dependencies`. If you use a dependency to only `import` things it can be required as a `devDependencies` as it is used to only build your app.
As it is already done that way, it's ok to leave it that way but we have to be sure that we don't install/bring unnecessary dependencies at runtime